spec/intl.html

```diff
@@ -0,0 +1,584 @@
+<!doctype html>
+<meta charset="utf8">
+
+<emu-clause id=sec-temporal-intl>
+  <h1>Integration with the Intl.DateTimeFormat interface</h1>
+
+  <emu-clause id="sec-datetimeformat-abstracts">
+    <h1>Abstract Operations For DateTimeFormat Objects</h1>
+
+    <emu-table id="table-datetimeformat-components">
+      <emu-caption>Components of date and time formats</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th>Internal Slot</th>
+            <th>Property</th>
+            <th>Values</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>[[Weekday]]</td>
+          <td>`"weekday"`</td>
+          <td>`"narrow"`, `"short"`, `"long"`</td>
+        </tr>
+        <tr>
+          <td>[[Era]]</td>
+          <td>`"era"`</td>
+          <td>`"narrow"`, `"short"`, `"long"`</td>
+        </tr>
+        <tr>
+          <td>[[Year]]</td>
+          <td>`"year"`</td>
+          <td>`"2-digit"`, `"numeric"`</td>
+        </tr>
+        <tr>
+          <td>[[Month]]</td>
+          <td>`"month"`</td>
+          <td>`"2-digit"`, `"numeric"`, `"narrow"`, `"short"`, `"long"`</td>
+        </tr>
+        <tr>
+          <td>[[Day]]</td>
+          <td>`"day"`</td>
+          <td>`"2-digit"`, `"numeric"`</td>
+        </tr>
+        <tr>
+          <td>[[Hour]]</td>
+          <td>`"hour"`</td>
+          <td>`"2-digit"`, `"numeric"`</td>
+        </tr>
+        <tr>
+          <td>[[Minute]]</td>
+          <td>`"minute"`</td>
+          <td>`"2-digit"`, `"numeric"`</td>
+        </tr>
+        <tr>
+          <td>[[Second]]</td>
+          <td>`"second"`</td>
+          <td>`"2-digit"`, `"numeric"`</td>
+        </tr>
+        <tr>
+          <td>[[TimeZoneName]]</td>
+          <td>`"timeZoneName"`</td>
+          <td>`"short"`, `"long"`</td>
+        </tr>
+      </table>
+    </emu-table>
+
+    <emu-clause id="sec-initializedatetimeformat" aoid="InitializeDateTimeFormat">
+      <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
+
+      <p>
+        The abstract operation InitializeDateTimeFormat accepts the arguments _dateTimeFormat_ (which must be an object), _locales_, and _options_. It initializes _dateTimeFormat_ as a DateTimeFormat object. This abstract operation functions as follows:
+      </p>
+
+      <emu-alg>
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. Let _options_ be ? ToDateTimeOptions(_options_, `"any"`, `"date"`).
```

Comment:
For Temporal types, we should default to having all options enabled, because we use a separate code path below to pick our subset.  Please re-work this to use the ToDateTimeOptions for Date but not for Temporal, and enable all fields for Temporal.  You might be able actually to move the Temporal resolution stuff into ToDateTimeOptions.